whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/organisation/OrgNameAndCountryAttrValidator.java

```diff
@@ -84,14 +92,19 @@ private boolean isReferencedByRsMaintainedResource(final RpslObject rpslObject)
         }
         return false;
     }
+    private boolean isAttributeModified(final AttributeType attrType, final RpslObject originalObject, final RpslObject updatedObject) {
+        final CIString originalAttrValue = originalObject.getValueOrNullForAttribute(attrType);
+        final CIString updatedAttrValue = updatedObject.getValueOrNullForAttribute(attrType);
 
-    private boolean orgNameDidntChange(final RpslObject originalObject, final RpslObject updatedObject) {
-        final CIString originalOrgName = originalObject.getValueOrNullForAttribute(AttributeType.ORG_NAME);
-        final CIString updatedOrgName = updatedObject.getValueOrNullForAttribute(AttributeType.ORG_NAME);
+        if(originalAttrValue == null && updatedAttrValue == null) {
+            return false;
+        }
+
+        if(originalAttrValue == null || updatedAttrValue == null) {
```

Comment:
Remove duplicate code

Reply:
May be a better way to handle this?

Reply:
Simplified the logic, CISTRING tostring method was the cause of null pointer exception

Reply:
You can simplify the logic like:
```
        return ((originalAttrValue != null && updatedAttrValue == null) ||     // removed
                    (originalAttrValue == null && updatedAttrValue != null) ||.    // added
                    !Objects.equals(originalAttrValue, updatedAttrValue));         // changed
```
which I confirmed with these cases before and afterwards (although the method is private):
```
    @Test
    public void test() {
        assertThat(isAttributeModified(AttributeType.COUNTRY, RpslObject.parse("organisation: ORG-RIEN1-RIPE"), RpslObject.parse("organisation: ORG-RIEN1-RIPE")), is(false));       // not present
        assertThat(isAttributeModified(AttributeType.COUNTRY, RpslObject.parse("organisation: ORG-RIEN1-RIPE"), RpslObject.parse("organisation: ORG-RIEN1-RIPE\ncountry: NL")), is(true));   // added
        assertThat(isAttributeModified(AttributeType.COUNTRY, RpslObject.parse("organisation: ORG-RIEN1-RIPE\ncountry: NL"), RpslObject.parse("organisation: ORG-RIEN1-RIPE")), is(true));   // removed
        assertThat(isAttributeModified(AttributeType.COUNTRY, RpslObject.parse("organisation: ORG-RIEN1-RIPE\ncountry: NL"), RpslObject.parse("organisation: ORG-RIEN1-RIPE\ncountry: DE")), is(true));   // changed
        assertThat(isAttributeModified(AttributeType.COUNTRY, RpslObject.parse("organisation: ORG-RIEN1-RIPE\ncountry: NL"), RpslObject.parse("organisation: ORG-RIEN1-RIPE\ncountry: nl")), is(false));  // not changed
    }
```

Reply:
So it's a bit better, but not hugely, so feel free to leave it as-is (I was just confused by the separate if statements)